techniques/system/rudder-service-postgresql/1.0/password-check.st

```diff
@@ -24,10 +24,10 @@ bundle agent system_rudder_postgresql(trackingkey, technique_name, db_pass) {
 
     any::
     # Array used to force content of the .pgpass file
-      "pgpass[1]" string => "${rudder_postgresql.host}";
+      #"pgpass[1]" string => "${rudder_postgresql.host}";
```

Comment:
I'm not sure it really makes sense to support a different db name if we don't support a different host

Reply:
i hope to add support of a different host - see https://issues.rudder.io/issues/19659